pkg/operation/common/types.go

```diff
@@ -450,6 +450,23 @@ const (
 	// RubyImageName is the name of the Ruby image.
 	RubyImageName = "ruby"
 
+	// AlicloudControllerManagerImageName is the name of the AlicloudControllerManager image.
+	AlicloudControllerManagerImageName = "alicloud-controller-manager"
+
+	// CSI Images
+
+	// CsiAttacherImageName is the name of csi attacher - https://github.com/kubernetes-csi/external-attacher
```

Comment:
Please capitalize CSI --> `CSIAttacherImageName` (same for others).

Reply:
Done